index.md

```diff
@@ -238,3 +238,67 @@ Our typical usages of AWS include:
 1. S3 for hosting uploaded files
 1. ElastiCache instances for hosting in-memory data stores
 1. OpsWorks for provisioning each if the size or complexity is large
+
+# Documentation
+
+Traditionally documentation has been a second-class citizen in most Rails project. This holds true just as much outside The Nerdery as internally. As innovators and leaders in our industry we should hold ourselves to a higher standard and therefore provide ample documentation on the products we produce. A great article on improving Rails based documentation published by Aaron Sumner can be found [here](http://everydayrails.com/2016/04/18/rails-documentation-practices.html) and many of the practices described below are pulled from his philosophy.
+
+## Start with the README
+Your application’s README is still its gateway, the first thing new developers will likely read when they access its source code. If your app’s README consists of the default given to every new Rails app, it’s time to change it.
+
+Your README should include:
+
+* A brief description of what the application does.
+* How to install dependencies, especially those that are more complex than a simple `bundle install`. For example, if I need to to install Redis or ImageMagick to work on the app, or configure environment variables to talk to external services, let me know up front.
+* How to run the application’s test suite, to verify that setup was successful. Again, note special external dependencies, like PhantomJS.
+* How to generate the docs, to get to a more detailed description of the application’s code.
+
+## Remove generated documentation from source control
+
+The HTML version of your Rails docs will need to be regenerated regularly, in order to keep up with the code they explain. Ignoring the doc/app directory makes sure that stale documentation doesn't stick around from commit to commit, and just makes commits cleaner in general. This is why it’s important to mention how to generate the docs in your README as they won’t be easily accessible, otherwise.
+
+## Write clear documentation
+
+* __Don't assume knowledge of the business__: If you've been working in the application for awhile, the names you've given your classes and methods may seem obvious, but to a newcomer they can be a large hurdle to overcome. Help explain ___why___ things are called what they are.
+* __Do assume knowledge of Rails__: The rule of thumb is to document ___why___, now ___how___. Document under the assumption that the reader can read Ruby code, or at least knows how to search Google or Stack Overflow to learn more about specific methods.
+
```

Comment:
Welcome to suggestions if folks have them :). If I think of more I'll certainly add them in.